hampusborgos/rme
# Fixes on 10.10

@dalkon
To test i put
![captura_de_tela-6](https://f.cloud.github.com/assets/5075051/2237575/64a285da-9bb3-11e3-99ff-0897b8453249.png)

10.3 see
![captura_de_tela-5](https://f.cloud.github.com/assets/5075051/2237577/718d1a12-9bb3-11e3-9dac-ff64accd4321.png)

10.1 see
![captura_de_tela-4](https://f.cloud.github.com/assets/5075051/2237578/786f38ec-9bb3-11e3-8242-7417050c5a61.png)

I ask you to test before...
